src/languages/en.ts

```diff
@@ -1627,34 +1627,60 @@ export default {
         hasBeenThrottledError: 'An error occurred while adding your bank account. Please wait a few minutes and try again.',
         hasCurrencyError: 'Oops! It appears that your workspace currency is set to a different currency than USD. To proceed, please set it to USD and try again.',
         error: {
-            youNeedToSelectAnOption: 'You need to select an option to proceed.',
-            noBankAccountAvailable: 'Sorry, no bank account is available.',
-            noBankAccountSelected: 'Please choose an account.',
-            taxID: 'Please enter a valid tax ID number.',
-            website: 'Please enter a valid website. The website should be in lowercase.',
-            zipCode: `Incorrect zip code format. Acceptable format: ${CONST.COUNTRY_ZIP_REGEX_DATA.US.samples}`,
-            phoneNumber: 'Please enter a valid phone number.',
-            companyName: 'Please enter a valid legal business name.',
-            addressCity: 'Please enter a valid city.',
-            addressStreet: 'Please enter a valid street address that is not a PO box.',
-            addressState: 'Please select a valid state.',
-            incorporationDateFuture: 'Incorporation date cannot be in the future.',
-            incorporationState: 'Please select a valid state.',
-            industryCode: 'Please enter a valid industry classification code. Must be 6 digits.',
-            restrictedBusiness: 'Please confirm company is not on the list of restricted businesses.',
-            routingNumber: 'Please enter a valid routing number.',
-            accountNumber: 'Please enter a valid account number.',
-            routingAndAccountNumberCannotBeSame: 'The routing number and account number cannot be the same.',
-            companyType: 'Please select a valid company type.',
-            tooManyAttempts: 'Due to a high number of login attempts, this option has been temporarily disabled for 24 hours. Please try again later or manually enter details instead.',
-            address: 'Please enter a valid address.',
-            dob: 'Please select a valid date of birth.',
-            age: 'Must be over 18 years old.',
-            ssnLast4: 'Please enter valid last 4 digits of SSN.',
-            firstName: 'Please enter a valid first name.',
-            lastName: 'Please enter a valid last name.',
-            noDefaultDepositAccountOrDebitCardAvailable: 'Please add a default deposit bank account or debit card.',
-            validationAmounts: 'The validation amounts you entered are incorrect. Please double-check your bank statement and try again.',
+            // eslint-disable-next-line rulesdir/use-periods-for-error-messages
```

Comment:
Why are we removing full stop. Can we just keep it?

Reply:
I'm going by this conversation from the original PR: https://github.com/Expensify/App/pull/44418/files#r1659334495 - I think we're moving to not keeping them in one liners.

Reply:
Thanks for linking it. So I think we need to change the rule rather than suppressing it, right?

Reply:
Sure, that's even better! Since I wasn't sure if that was already being worked on or not, or when that would be complete, I went with Manan's suggestion.

Reply:
If it's a quick fix, let me know and I can remove these comments.

Reply:
The code is here https://github.com/Expensify/eslint-config-expensify/blob/92a1f5fef073b21d940f55577f18095cc9d2b7ac/eslint-plugin-expensify/use-periods-for-error-messages.js#L17-L29
We can update it to use period if there is only more than one line, as discussed in https://github.com/Expensify/App/pull/44418/files#r1659334495

Reply:
you can clone that repo and develop locally following these instructions
https://github.com/Expensify/eslint-config-expensify#eslint-config-expensify

Reply:
Cloned it via https and I'm seeing this error now, should I re-clone it another way or is there a way to authenticate?

`eslint-config-expensify git:(cm-update-period-rule) git po cm-update-period-rule
Username for 'https://github.com': carlosmiceli
Password for 'https://carlosmiceli@github.com':
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/Expensify/eslint-config-expensify.git/'`

Reply:
You can try ssh
`git clone git@github.com:Expensify/eslint-config-expensify.git`

Reply:
There!: https://github.com/Expensify/eslint-config-expensify/pull/105